Content.Server/_Umbra/Atmos/EntitySystems/SpontaneousCombustionSystem.cs

```diff
@@ -0,0 +1,127 @@
+using Content.Server.Atmos.Components;
+using Content.Shared.Atmos;
+using Content.Shared.Inventory;
+using Content.Shared.Inventory.Events;
+using Robust.Shared.Containers;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Content.Server.Atmos.EntitySystems;
```

Comment:
```suggestion
namespace Content.Server._Umbra.Atmos.EntitySystems;
```
If you update all the namespaces, you may need to add additional `using` statements, just be aware of that if things break as you update.

Reply:
fixed